8Knot/queries/prs_query.py

```diff
@@ -37,76 +33,51 @@ def prs_query(self, repos):
 
     query_string = f"""
                     SELECT
-                        r.repo_id as id,
+                        r.repo_id,
                         r.repo_name,
                         pr.pull_request_id AS pull_request,
                         pr.pr_src_number,
+                        left(pr.pr_augur_contributor_id::text, 15) as cntrb_id,
+                        -- values are timestamp not timestamptz
                         pr.pr_created_at AS created,
                         pr.pr_closed_at AS closed,
-                        pr.pr_merged_at  AS merged
+                        pr.pr_merged_at AS merged
                     FROM
-                        repo r,
-                        pull_requests pr
+                        augur_data.repo r,
```

Comment:
why the addition of augur_data to all queries?

Reply:
Because I didn't know how to set the desired schema in the psycopg2 driver configuration. I'll work on that to remove the direct references to augur_data.

Reply:
added the schema as a search_path parameter in the psycopg2.connect call, so we don't need 'augur_data' in the queries anymore